src/verifier.py

```diff
@@ -22,9 +22,35 @@ def verify_ballot_consistency(self) -> bool:
         the "Proving Equality of Arrays of Vote Values" procedure described
         in Section II-F.
         """
-        # TODO
-        # Parse SBB
-        # Use (t, -t) and prior commitments posted to verify
+        sbb_contents = self._sbb.get_sbb_contents()
+        
+        # First, validate the commitment consistency with the initial vote lists and final vote lists.
```

Comment:
Is this just a sanity check? I don't think it is a part of the original method nor does it seem mandatory, if the equality check using `t` passes then that proves this as well.

Reply:
Input openings, output openings, and t values all need to be checked -- at least that's how I understood it. I also peeked at the rivest implementation and it validates all three as well